editor/src/components/canvas/__snapshots__/ui-jsx-canvas.spec.tsx.snap

```diff
@@ -21840,7 +22340,7 @@ export var storyboard = (
       },
     },
   },
-  "eee/fff:aaa/ddd": Object {
+  "eee/fff:aaa/bbb": Object {
```

Comment:
This is interesting. Something from here onwards has been parsed or walked completely differently :thinking:

Reply:
`eee/fff:aaa/bbb` was previously not in the spy data, and now it is!

Reply:
the spy data for aaa/ddd is intact, it has been moved lower though.